tests/builtins/test_abs.py

```diff
@@ -8,7 +8,8 @@ class NotAbsLike:
                 pass
             x = NotAbsLike()
             print(abs(x))
-            """)
+            print('Done.')
+            """, exits_early=True)
```

Comment:
I can go through all the tests for `print('Done.')` and swap them out for a raise/except block, but is doing `print(err)` to let the test suite match strings sufficient to check that it's the correct exception?